tests/run_testsuite.cmake

```diff
@@ -540,7 +547,10 @@ IF("${_res}" STREQUAL "0")
     IF(COVERAGE)
       CREATE_TARGETDIRECTORIES_TXT()
       MESSAGE("-- Running CTEST_COVERAGE()")
-      CTEST_COVERAGE()
+      #CTEST_COVERAGE()
+      FILE(DOWNLOAD "https://codecov.io/bash" "${CMAKE_CURRENT_BINARY_DIR}/tests/codecov-bash")
+      EXECUTE_PROCESS(COMMAND bash "${CMAKE_CURRENT_BINARY_DIR}/tests/codecov-bash"
+                              "-t ac85e7ce-5316-4bc1-a237-2fe724028c7b" "-x '${GCOV_COMMAND}'")
```

Comment:
Is there any other way than introducing a remote code execution in the `run_testsuite.cmake` script?

Reply:
Not that I know of. The recommended (only?) way for uploading the coverage results is running this bash script.

Reply:
I am in favor of the latter approach - require to manually download and verify the script (and simply do not submit to codecov if the script is absent).